erpnext_germany/erpnext_germany/doctype/business_letter/business_letter.json

```diff
@@ -191,12 +197,41 @@
    "options": "Company",
    "print_hide": 1,
    "remember_last_selected_value": 1
+  },
+  {
+   "fieldname": "author",
+   "fieldtype": "Link",
+   "in_list_view": 1,
+   "label": "Author",
+   "options": "Employee",
+   "print_hide": 1,
+   "remember_last_selected_value": 1,
+   "reqd": 1
```

Comment:
This should not be mandatory because it improves backwards compatibility and making it mandatory is an easy customization.